safe/impact_functions/core.py

```diff
@@ -143,3 +143,21 @@ def has_no_data(layer_data):
     :rtype: bool
     """
     return numpy.isnan(numpy.sum(layer_data))
+
+
+def get_key_for_value(value, value_map):
+    """Obtain the key of a value from a value map.
+
+    :param value: The value mapped to a key in value_map.
+    :type value: int, str, float
+
+    :param value_map: A value mapping.
+    :type value_map: dict
+
+    :returns: A key for the value.
+    :rtype: str
+    """
+    for key, values in value_map.iteritems():
```

Comment:
Hmm, I thought we need to change the data structure for the value mapping. It should be real value mapped to single class. E.g.
```python
value_map = {
'KRB1' : 'low', 
'Kawasan Rawan Bencana 1' : 'low', 
'KRB2' : 'medium', 
'Kawasan Rawan Bencana 2' : 'medium', 
'KRB3' : 'high', 
'Kawasan Rawan Bencana 3' : 'high}
```

But for now, we will use the current data structure, and add some notes in this function like your suggestion.